reconcile/test/test_dashdotdb_dora.py

```diff
@@ -0,0 +1,292 @@
+from datetime import datetime
+from unittest.mock import MagicMock
+
+import yaml
+from pytest_mock import MockerFixture
+
+from reconcile.dashdotdb_dora import (
+    AppEnv,
+    Commit,
+    DashdotdbDORA,
+    Deployment,
+    RepoChanges,
+    SaasTarget,
+    SummaryEntry,
+    SummaryKey,
+)
+
+
+def test_get_repo_ref_for_sha(mocker: MockerFixture):
+    mocker.patch("reconcile.dashdotdb_dora.DashdotdbDORA.__init__").return_value = None
```

Comment:
After refactoring your `DashdotdbDORA` class, the `__init__` mocks should no longer be needed. You can use `pytest.fixture`'s to get your dependencies required, e.g., GitLabApi. I think you'll find examples in our existing tests.